ui/appui-react/src/appui-react/messages/ActivityMessage.tsx

```diff
@@ -26,37 +34,110 @@ export interface ActivityMessageProps {
 
 /** Activity Message React component
  * @public
+ * @deprecated
  */
 export function ActivityMessage(props: ActivityMessageProps) {
   const messageDetails = props.activityMessageInfo.details;
   const [percentCompleteLabel] = React.useState(UiFramework.translate("activityCenter.percentComplete"));
   const [cancelLabel] = React.useState(UiCore.translate("dialog.cancel"));
 
   return (
-    <ToastPresentation
-      category="informational"
-      hasCloseButton={true}
-      onClose={props.dismissActivityMessage}
-      content={
-        <MessageLayout
-          buttons={(messageDetails && messageDetails.supportsCancellation) &&
-            <MessageHyperlink onClick={props.cancelActivityMessage}>{cancelLabel}</MessageHyperlink>
-          }
-          progress={(messageDetails && messageDetails.showProgressBar) &&
+    <Message // eslint-disable-line deprecation/deprecation
```

Comment:
Just revert the UI as it was previously to remove `ToastPresentation` usage. `ActivityMessage` itself is not used anywhere